assets/scss/02-molecules/_labelled-list.scss

```diff
@@ -1,4 +1,9 @@
 .ma__labelled-list {
+  margin-top: 30px !important;
+
+  & + .ma__rich-text {
+    margin-top: 30px !important;

```

Comment:
same as above @clairesunstudio  --> do we want to consistently use rem for margins? - also !Important